client/shipping-label/views/history.js

```diff
@@ -0,0 +1,79 @@
+import React, { PropTypes } from 'react';
+import CalypsoI18n from 'i18n-calypso';
+import Gridicon from 'components/gridicon';
+import formatDate from 'lib/utils/format-date';
+import { translate as __ } from 'lib/mixins/i18n';
+import { sprintf } from 'sprintf-js';
+const LabelHistory = ( { labelsHistory } ) => {
+	if ( ! labelsHistory || ! labelsHistory.length ) {
+		document.getElementById( 'woocommerce-order-label-history' ).style.display = 'none';
+		return null;
+	}
+	document.getElementById( 'woocommerce-order-label-history' ).style.display = 'block';
```

Comment:
This has to hide the whole meta box, which is not controlled by React, but by PHP. I could use a class, but it is easier and more cross-browser compatible to just set the CSS `display` directly.